st2common/st2common/metrics/metrics.py

```diff
@@ -0,0 +1,162 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
```

Comment:
Maybe call that module base for consistency with some other packages? In any case, not a blocker :)